en_US/network/crl.md

```diff
@@ -0,0 +1,34 @@
+### CRL Check
+
+{% emqxce %}
+
+Since EMQX 5.0.22, Certification Revocation List (CRL) Check is supported for MQTT SSL listeners.  Note that those do not include Secure WebSocket nor QUIC listeners: only listeners of type `ssl` support this feature.
+
+{% endemqxce %}
+
+{% emqxee %}
+
+Since EMQX 5.0.3, Certification Revocation List (CRL) Check is supported for MQTT SSL listeners.  Note that those do not include Secure WebSocket nor QUIC listeners: only listeners of type `ssl` support this feature.
+
+{% endemqxee %}
+
+With this feature enabled, EMQX will attempt to verify if connecting client certificates are not revoked according to the CRL Distribution Point described in the client's certificate, and deny connection to revoked client certificates during the SSL/TLS handshake phase of the connection.  Note that the CRL itself must contain the ["Issuing Distribution Point" extension](https://www.rfc-editor.org/rfc/rfc3280#section-5.2.5) in order for the revocation check to be enforced.
+
+In order to enable this feature, we need to both enable the corresponding option in the listener and also set the `verify` option of the listener to `verify_peer`, so that the client must be checked against the CRL.
+
+Example configuration to enable CRL Check:
+
+
+```hcl
+listeners.ssl.default {
+  bind = "0.0.0.0:8883"
+  max_connections = 512000
+  ssl_options {
+    keyfile = "/etc/emqx/certs/server.key"
+    certfile = "/etc/emqx/certs/server.pem"
+    cacertfile = "/etc/emqx/certs/ca.pem"
+    verify = verify_peer
+    enable_crl_check = true
```

Comment:
Need to confirm, 4.4.x has multiple configuration items

```bash
   enable_crl_check = true
    crl_cache_urls = http://my.crl.server/intermediate.crl.pem, http://my.other.crl.server/another.crl.pem
    crl_cache_refresh_interval = 15m
    crl_cache_http_timeout = 15s
```

Reply:
In 5.0, we don't explicitly set the cache urls anymore.  The caching mechanism is hidden from the user.  There are advanced configs that configure the cache size, refresh timeout and HTTP timeout, but those are hidden.

Reply:
@thalesmg 
Does hidden mean that it is not exposed to the user by default, but if configured, it can still work?

@LenaLenaPan For this type of configuration I think it could be identified in the documentation as an "advanced option"?

Reply:
@wivwiv Yes.  It won't be return in API returns nor listed in `schema.json` and auto-generated docs.  But if such config is sent via the API or set in config files, it'll take effect.

Just to be clear: `crl_cache_urls` is not even hidden, it doesn't exist in 5.0.